ui/src/album/AlbumListActions.js

```diff
@@ -57,6 +59,9 @@ const AlbumListActions = ({
           <ViewHeadlineIcon fontSize="inherit" />
         </Button>
       </ButtonGroup>
+      {isDesktop && !albumView.grid ? (
+        <ToggleFieldsMenu resource="album" />
```

Comment:
Not sure if this is the best place for Albums. It makes the other buttons move.... Can't think of a better solution, though...

https://user-images.githubusercontent.com/331353/114949165-67a4e780-9e1e-11eb-9d34-b1fe124cbddb.mov

Reply:
Ahh yes. The other option was to keep the menu icon there on both views. Maybe we could ask the opinion of others as well.